src/utils/focusTrapComponent.ts

```diff
@@ -95,3 +100,19 @@ export function deactivateFocusTrap(component: FocusTrapComponent): void {
 export function focusFirstTabbable(component: FocusTrapComponent): void {
   tabbable(component.focusTrapEl, tabbableOptions)[0]?.focus();
 }
+
+/**
+ * Helper to update the element(s) that are used within the FocusTrap component.
+ *
+ * @param {FocusTrapComponent} component The FocusTrap component.
+ * @example
+ * const modal = document.querySelector("calcite-modal");
+ * const input = document.createElement("calcite-input");
+ * content.appendChild(input);
+ * await input.componentOnReady();
+ * await modal.updateFocusTrapElements();
+ * requestAnimationFrame(() => input.setFocus());
+ */
```

Comment:
@macandcheese do we recieve the `example` tag via `docs-json`? It would be nice to add examples to the API ref table. Maybe something like the "show code snippet" action you have in the component preview. We get a lot of feedback about a lack of javascript examples, so this could be a good solution.

Thanks for adding it @driskull!